fast64_internal/oot/oot_utility.py

```diff
@@ -602,3 +633,266 @@ def oot_utility_register():
 def oot_utility_unregister():
 	for cls in reversed(oot_utility_classes):
 		unregister_class(cls)
+
+def getActorParameter(object, field, shift, mask):
+	'''Returns the actor parameter with the shifts'''
+	attr = getattr(object, field, '0x00')
+	if attr != '0x00' and attr is not None:					
+		if isinstance(attr, str):
+			if shift != '0':
+				return f"(({attr} << {shift}) & {mask}) | "
```

Comment:
I think the ` | ` at the end doesn't fit the function

Removing it from here and handling `|`ing the parts in `getActorString` would be cleaner, and you could remove `params = params.rstrip().rstrip('|').rstrip()`

Reply:
to get rid of `params = params.rstrip().rstrip('|').rstrip()`, I was thinking about collecting params parts into a list in `getActorString` and then using `" | ".join()`
it would also help with not having to append `" | "` when the params part is empty